judgments/feeds.py

```diff
@@ -158,7 +159,17 @@ def item_title(self, item):
         return item.name
 
     def item_guid(self, item) -> str:
-        return "https://caselaw.nationalarchives.gov.uk/id/" + item.uri
+        fclid = item.identifiers.of_type(FindCaseLawIdentifier)
+
+        if not fclid:
+            msg = f"No FindCaseLawIdentifier for search result {item}"
+            raise RuntimeError(msg)
+
+        if len(fclid) > 1:
+            msg = f"Multiple FindCaseLawIdentifiers for search result {item}"
+            raise RuntimeError(msg)
+
+        return "https://caselaw.nationalarchives.gov.uk/id/" + fclid[0].url_slug
```

Comment:
I don't think this should be the slug, since the whole point of slugs is to provide a shorthand version. Maybe we should decide on some ID namespaces (eg `/id/work/` or `/id/w-`)?